src/DynamoCore/Engine/CodeCompletion/CodeCompletionServices.cs

```diff
@@ -69,7 +69,7 @@ internal IEnumerable<CompletionData> GetCompletionsOnType(string code, string st
                 }
             }
 
-            return members?.Select(x => CompletionData.ConvertMirrorToCompletionData(x));
+            return members?.Distinct(new StaticMirrorNameComparer()).Select(x => CompletionData.ConvertMirrorToCompletionData(x));
```

Comment:
UI change :

`GetCompletionsOnType` shows autocomplete for all members (methods, properties, constructors) of a class. It used to show all members from all base classes (even duplicates - overloads or hidden members).

Using `Distinct(new StaticMirrorNameComparer())` we will filter out duplicate member names.

Autocomplete on function calls (ex. `classX.Func(` ) will show all members except hidden ones.